internal/db/schema/migrations/oss/postgres/99/03_worker_authentication.up.sql

```diff
@@ -16,6 +16,23 @@ values
   ('next'),
   ('current');
 
+create table worker_auth_ca(
+   private_id text primary key
+     constraint only_roots_id_allowed
+       check (
+           private_id in ('roots')
+         ),
+   version wt_version
+);
+
+create trigger
+  update_version_column
+  after update of version on worker_auth_ca
+  for each row execute procedure update_version_column();
+
+comment on table worker_auth_ca is
+  'worker_auth_ca is a one-row versioned table used for locking for certificate rotation on the worker_auth_ca_certificate table.';
+
```

Comment:
should `worker_auth_ca`, `worker_auth_ca_certificate`, `worker_auth_authorized`, and `worker_auth_certificate_bundle` have immutable triggers?